src/bin/validator.rs

```diff
@@ -38,7 +38,7 @@ struct CliOpts {
     database_url: String,
 
     /// Listen address for the server
-    #[clap(short, long, env, default_value = "127.0.0.1:10000")]
+    #[clap(short, long, env, default_value = "127.0.0.1:1234")]
```

Comment:
How comes this was changed?

Reply:
The bundlr-node default port is already 10000, so I commited this.